install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1041,19 +1041,44 @@ hubble:
   #     enabled: true
   #     http:
   #       urlQuery: true
+  #       headers:
+  #         allow:
+  #           - traceparent
+  #           - tracestate
+  #           - Cache-Control
+  #         deny:
+  #           - Authorization
+  #           - Proxy-Authorization
```

Comment:
```suggestion
```
Could we keep only the allow list in this example to make the intent clear?

Reply:
Ah, right that's a leftover. Thanks!